OpenEXR/IlmImfTest/testFutureProofing.cpp

```diff
@@ -1241,7 +1241,9 @@ modifyType (bool modify_version)
             
             
             //length of attribute
-            fread(&length,4,1,f);
+            size_t nr = fread (&length,4,1,f);
+            if ( nr != 1 )
+                throw std::runtime_error ("unable to read length of attribute");
```

Comment:
To be consistent with other usage in IlmImfTest, this should throw an Iex::IoExc.

Reply:
yeah, fixed both of these up.